app/default_overrides.go

```diff
@@ -75,12 +79,33 @@ func (stakingModule) DefaultGenesis(cdc codec.JSONCodec) json.RawMessage {
 	params := stakingtypes.DefaultParams()
 	params.UnbondingTime = appconsts.DefaultUnbondingTime
 	params.BondDenom = BondDenom
+	params.MinCommissionRate = sdk.NewDecWithPrec(5, 2) // 5%
 
 	return cdc.MustMarshalJSON(&stakingtypes.GenesisState{
 		Params: params,
 	})
 }
 
+// stakingModule wraps the x/staking module in order to overwrite specific
+// ModuleManager APIs.
+type slashingModule struct {
+	slashing.AppModuleBasic
+}
+
+// DefaultGenesis returns custom x/staking module genesis state.
+func (slashingModule) DefaultGenesis(cdc codec.JSONCodec) json.RawMessage {
+	params := slashingtypes.DefaultParams()
+	params.MinSignedPerWindow = sdk.NewDecWithPrec(5, 2) // 5%
+	params.SignedBlocksWindow = 15000
```

Comment:
https://github.com/celestiaorg/celestia-app/blob/4e62d2d3660e8ad5b6115348bed5aa6a57cf93fc/specs/src/specs/params.md?plain=1#L44 need to be updated

Reply:
With the above 5% that's about missing slightly more than 3 hours woth of blocks to get slashed 1%. 
This could be a bit too strict tbh. What do other networks chose here?

Reply:
updated in https://github.com/celestiaorg/celestia-app/pull/2417/commits/0d6fd942bd3b34d453a5ced73de456be9b264be2 to the latest values per sync discussion